packages/widgets/bettinggame/src/components/TournamentWidget.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types'
+import EntityDetailApp from 'tocco-entity-detail/src/main'
+
+const TournamentWidget = props => {
+  const {tournamentKey, formBase} = props
+  return (
+    <EntityDetailApp
+      id={`detail_${formBase}_${tournamentKey}`}
+      entityName="Tournament"
+      entityId={tournamentKey}
+      formName={formBase}
+      mode="update"
+    />
+  )
+}
+
+TournamentWidget.propTypes = {
+  tournamentKey: PropTypes.number,
```

Comment:
question(blocking): Are you sure that this is of type `number` and not of type `string`?